client/src/state/editor/addElementMutation.js

```diff
@@ -0,0 +1,42 @@
+import { graphql } from 'react-apollo';
+import gql from 'graphql-tag';
+
+// GraphQL query for deleting a specific block
+const mutation = gql`
+mutation AddBlock($className: String!, $elementalAreaID: ID!, $afterElementID: ID) {
+  addElementToArea(
+    ClassName: $className,
+    ElementalAreaID: $elementalAreaID,
+    AfterElementID: $afterElementID
+  ) {
+    ID,
+    Sort,
+    IsPublished,
+    IsLiveVersion,
```

Comment:
These commas aren't needed. I actually thought GraphQL kicks up a fuss but apparently not 🤔 . Additionally you're doing a refetch so you can technically omit all return values. 

Alternatively you can update the return values to match `readBlocksForPage` and then you can do an update on the store after the values are returned. See the sort mutation: https://github.com/dnadesign/silverstripe-elemental/blob/master/client/src/state/editor/sortBlockMutation.js#L28. The `optimisticResponse` is not relevant because you can't assume the response, but you should be able to splice your item into the known list of blocks.

Essentially here you're reading the cached response of a GraphQL query and updating that response. Apollo will automatically update consumers of that query.

This will drop a request to the server (the refetch of the query)

Reply:
So turns out commas are OK.
So are line returns.
So are spaces.
So are combinations of all of the above 😑

Reply:
They might be OK but in lieu of a GraphQL query "coding standard" we should at least aim for uniformity among the queries in core and the wider ecosystem (no commas)

Reply:
OK, so we looked at this together, however as the `pageId` is required we (@ScopeyNZ and I) thought it better to wait for the refactor from `getElementsForPage` to become `getElementsForArea` - we already have the `ElementalAreaID`, and re-inserting the PageID to filter all the way through all areas seems... unnecessary (read: tech debt that we'd have to remove later).